iOS/UI/Shared/ReplySummaryView.swift

```diff
@@ -55,6 +55,7 @@ class ReplySummaryView: BaseView {
     var didTapViewReplies: CompletionOptional = nil
     var didSelectSuggestion: ((SuggestedReply) -> Void)? = nil
     var didSelectEmoji: ((String) -> Void)? = nil
+    private var replyCount = 0
```

Comment:
This was really bothering me so I added this little check. This won't account for edits, deletes etc but I think will cover 99% of the use case of new ones being added.

Reply:
I'd rather store an int than an array of message objects...